devtools.html

```diff
@@ -0,0 +1,10 @@
+<script type="module">
+    import * as aimless from "./src/aimless.js";
+    import * as utils from "./src/utils.js";
+    Object.assign(window, { ...aimless, ...utils });
+</script>
+<pre>
+// just type any code in F12/DevTools like this:
+floatRange(10, 20); // e.g. 17.68330329779974
+defaultEngine(); // e.g. 0.7867804299919436
+</pre>
```

Comment:
there's a similar file for manual testing here: https://github.com/ChrisCavs/aimless.js/blob/main/test/manual.html

Reply:
Thank you, didn't see that one. It just has three problems rendering it less useful for testing:
- no access to all exported functions:
![image](https://github.com/ChrisCavs/aimless.js/assets/5236548/9df68fc5-fe94-4dc0-b42b-c96b76724266)
- requires a build step, which slows down testing and debugging
- requires to prefix everything with `aimless`, deviating from modern syntax or how you would write code in any normal setting (either CJS or MJS)